llvm/include/llvm/TargetParser/Triple.h

```diff
@@ -769,7 +769,7 @@ class Triple {
   bool isOSGlibc() const {
     return (getOS() == Triple::Linux || getOS() == Triple::KFreeBSD ||
             getOS() == Triple::Hurd) &&
-           !isAndroid() && !isMusl();
+           !isAndroid() && !isMusl() && getEnvironment() != Triple::PAuthTest;
```

Comment:
Please add comment that pauthtest as of now implies not particular libc implementation